webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/facts/nodes/NodeFactRepository.scala

```diff
@@ -155,6 +157,12 @@ class GitNodeFactRepository(
   override val fileFormat: String = "1"
   val commiter = new PersonIdent("rudder-fact", "email not set")
 
+  if (actuallyCommit) {
+    NodeLogger.info("Nodes changes will be historized in Git in /var/rudder/fact-repository/nodes")
```

Comment:
This message should use the variable to fact-repository path (gitRepo rootdir ? )